Controller/RegistrationController.php

```diff
@@ -96,7 +96,7 @@ public function registerAction(Request $request)
     /**
      * Tell the user to check his email provider.
      */
-    public function checkEmailAction()
+    public function checkEmailAction(Request $request)
```

Comment:
Can someone check have symfony behaves when you add `Request $request = null` ? Is the `$request` filled? If yes, this should be closed (or at least shrinked).